src/ray/object_manager/object_manager.cc

```diff
@@ -90,6 +94,10 @@ void ObjectManager::NotifyDirectoryObjectAdd(const ObjectInfoT &object_info) {
   local_objects_[object_id] = object_info;
   ray::Status status =
       object_directory_->ReportObjectAdded(object_id, client_id_, object_info);
+}
+
+void ObjectManager::HandleUnfulfilledPushRequests(const ObjectInfoT &object_info) {
+  ObjectID object_id = ObjectID::from_binary(object_info.object_id);
```

Comment:
Minor, but I think this can be `const ObjectID`. This comment applies in a ton of places (we haven't been very religious about it, but I think it is preferable to do that).